src/argilla/server/schemas/v1/suggestions.py

```diff
@@ -32,6 +32,10 @@ class SuggestionCreate(BaseSuggestion):
     pass
 
 
+class SuggestionCreateWithRecordId(BaseSuggestion):
+    record_id: UUID
```

Comment:
If this schema is not used in any endpoint, we should move from here. Not sure if as part of the `contexts`, or related to the `models` modules, as we have for questions. Otherwise, could create confusion regarding the layer separation.